packages/hint/src/lib/cli/analyze.ts

```diff
@@ -3,8 +3,9 @@ import { URL } from 'url';
 import * as path from 'path';
 
 import * as async from 'async';
-import { default as ora, Ora }  from 'ora';
+import { default as ora, Ora } from 'ora';
 const boxen = require('boxen'); // `require` used because `boxen` exports a function
+
```

Comment:
Nit: remove this extra empty line that you added.

Reply:
I can't remove that line, because after a  `const` we need an empty line :P.
I move the const below in the code, so all the imports are together :)

Reply:
I need to update how we import `boxen` and remove the types. I've created https://github.com/webhintio/hint/issues/2036 to track it.